specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/network/group/client.sent.write.abort.before.coordinator.connected/client.rpt

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
What does `connected` mean here?

Suggest renaming scenario to `client.sent.write.abort.before.coordinator.response`.

Reply:
Makes sesnse